ucm2/Intel/avs/avs_dmic/Google-Atlas-1.0-HiFi.conf

```diff
@@ -0,0 +1,16 @@
+SectionDevice."Mic" {
+	Comment "Digital Microphone"
+
+	Value {
+		CapturePCM "hw:${CardId},2"
+		CaptureChannels 4
+		# While NHLT only has entry for 4 channel DMIC, only 2 channels
+		# record anything reasonable, so for Front End to work properly
+		# remap two working channels onto 2 broken ones.
+		CaptureChannel0 0
```

Comment:
Unfortunately, the audio servers has not implemented this yet. It's preferable to use `ucm2/common/pcm/split.conf` macros to use alsa-lib's route plugin.

Reply:
Now, that's weird, if I remember right, it worked, but yes, I don't see it used anywhere in code for alsa-lib, pulseaudio or pipewire :/  Will check it again then.

Reply:
Changed Atlas to link to base 4 channel DMIC variant.